packetery/src/Packetery/Plugin.php

```diff
@@ -264,9 +264,26 @@ public function render_email_footer(): void {
 	 * Enqueues javascript files for administration.
 	 */
 	public function admin_enqueue_scripts(): void {
-		wp_enqueue_script( 'live-form-validation', plugin_dir_url( $this->main_file_path ) . 'public/libs/live-form-validation/live-form-validation.js', array(), '2.0-dev', false );
-		wp_enqueue_script( 'admin-country-carrier', plugin_dir_url( $this->main_file_path ) . 'public/admin-country-carrier.js', array(), self::VERSION, true );
-		wp_enqueue_style( 'packetery-admin-styles', plugin_dir_url( $this->main_file_path ) . 'public/admin.css', array(), self::VERSION );
+		wp_enqueue_script(
```

Comment:
Než neustále opakovat velmi podobný kód a každý soubor uvádět 2x,  nebylo by hezčí vytvořit pole se seznamem načítaných souborů a toto pole pak zpracovat ?

Reply:
To pole je nepraktické. Metoda je lepší. Viz poslední změny.